tests/next_tests/unit_tests/conftest.py

```diff
@@ -35,18 +35,6 @@
 import next_tests
 
 
-@pytest.fixture(
-    params=[
-        transforms.LiftMode.FORCE_INLINE,
-        transforms.LiftMode.USE_TEMPORARIES,
-        transforms.LiftMode.SIMPLE_HEURISTIC,
-    ],
-    ids=lambda p: f"lift_mode={p.name}",
-)
-def lift_mode(request):
-    return request.param
-
-
 OPTIONAL_PROCESSORS = []
 if dace_iterator:
     OPTIONAL_PROCESSORS.append((next_tests.definitions.OptionalProgramBackendId.DACE_CPU, True))
```

Comment:
Should we enable `run_dace_cpu_with_temporaries` and `run_dace_gpu_with_temporaries`, which are already defined in `src/gt4py/next/program_processors/runners/dace.py`?

Reply:
- GPU is not enabled because in itir tests we don't (always) use the test infrastructure that uses the proper allocators.
- If you like you can try adding the DaCe with temporaries in a separate PR. It's not too important from my point of view because we shift focus on the fieldview->DaCe backend.